### ISSUE TITLE
Backup Automático

### ISSUE THREAD

**Issue by Altamir10** [vaamonde/pfsense#1]
Professor,
Para restaurar esta backup em outro hardware e se copiar a chave

**Comment by vaamonde** [comment]
Sim, conforme descrito na documentação oficial: https://docs.netgate.com/pfsense/en/latest/backup/restore.html - https://docs.netgate.com/pfsense/en/latest/backup/restore-during-install.html - https://docs.netgate.com/pfsense/en/latest/backup/restore-different-version.html - https://docs.netgate.com/pfsense/en/latest/backup/remote-backup.html - esse procedimento é possível, mais recomendo mesmo tendo o Backup ACB ter o backup manual, fazer o processo de transição/migração manualmente.
